pkg/store/sqlite/auth.go

```diff
@@ -223,6 +224,31 @@ func (s *Store) DeleteUser(ctx context.Context, userID ulid.ULID) (err error) {
 	return tx.Commit()
 }
 
+const lookupRoleSQL = "SELECT * FROM roles WHERE title like :role LIMIT 1"
```

Comment:
Note that `like` in sqlite3 is case insensitive, for postgres we would have to use `ilike`